site/src/xServices/templateVersionEditor/templateVersionEditorXService.ts

```diff
@@ -235,6 +276,12 @@ export const templateVersionEditorMachine = createMachine(
         publishingError: (_, event) => event.data,
       }),
       clearPublishingError: assign({ publishingError: (_) => undefined }),
+      assignMissingVariables: assign({
```

Comment:
This is the request to the backend, and it shouldn't receive `null`s:

<img width="1137" alt="Screenshot 2023-04-13 at 09 11 48" src="https://user-images.githubusercontent.com/14044910/231685858-94397c22-f61e-49a4-b275-62aa7c744f3e.png">

Reply:
Hm.... interesting, I will take a look into that.

Reply:
Confirmed it's fixed